be/src/runtime/runtime_filter_mgr.h

```diff
@@ -68,7 +72,8 @@ class RuntimeFilterMgr {
                          const TQueryOptions& options, int node_id = -1);
 
     // update filter by remote
-    Status update_filter(const PPublishFilterRequest* request, const char* data);
+    Status update_filter(const PPublishFilterRequest* request,
+                         butil::IOBufAsZeroCopyInputStream* data);
```

Comment:
```suggestion
                         const butil::IOBufAsZeroCopyInputStream* data);
```

it is better add const, because we can not modify attachment data.

Reply:
`IOBufAsZeroCopyInputStream.Next` is a non-const method so we can't make it const here